Electron/Core/RoveProtocol/Rovecomm.ts

```diff
@@ -9,6 +10,13 @@ const dgram = require("dgram")
 const net = require("net")
 const EventEmitter = require("events")
 
+interface TCPSocket {
+  RCSocket: Socket
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
See if you could remove this disable. Any should be okay and just be a yellow warning.